.github/workflows/lerna-publish.yml

```diff
@@ -1,14 +1,22 @@
 name: Publish NPM Packages
 
 on:
+  release:
+    types: published
+
   workflow_dispatch:
     inputs:
       ref:
         description: 'commit/tag/branch to release from'
         required: true
 
 jobs:
-  release:
+  publish:
+    if: >
+      (github.event.inputs) ||
+      (startsWith(github.event.release.tag_name, 'sdk-codegen-all') &&
+      !github.event.release.draft &&
+      !github.event.release.prerelease)
```

Comment:
this whole block is copied from the python publish: it just means "either use the form input from the manual workflow run OR use the tag as long as it's not draft or pre-release"

Reply:
it also means that now that `lerna publish` will run automatically after the Release PR is merged - I think we're ready for that